buildSrc/src/main/groovy/nva.publication.api.java-conventions.gradle

```diff
@@ -43,7 +43,7 @@ tasks.named('test') {
     useJUnitPlatform {
         excludeTags "RemoteTest", "integrationTest", "KarateTest"
     }
-    failFast = true
+    failFast = false
```

Comment:
Can probably turn this off again, but it was useful for finding legitimate contra flaky tests